swap/swap.go

```diff
@@ -85,27 +85,37 @@ func NewParams() *Params {
 	}
 }
 
-// newLogger returns a new logger
-func newLogger(logpath string) log.Logger {
-	swapLogger := log.New("swaplog", "*")
+// newSwapLogger returns a new logger for standard swap logs
+func newSwapLogger(s *Swap) log.Logger {
+	swapLogger := log.New("swaplog", "*", "selfAddress", s.owner.address)
```

Comment:
This swaplog should be refactored by it's actions, and not by a prefix swaplog
But I will be addressed (no pun intended) in another PR, derived from #1788

It's ok for now.

Reply:
i recommend you trim the address in a similar way we construct the loggers in `stream`:
`logger:         log.New("base", hex.EncodeToString(baseKey)[:16], "peer", peer.ID().String()[:16]),`
it makes the logs easier to read and the first 16 chars provide enough variance to distinguish between nodes

Reply:
very happy to read this, since it was what we agreed on doing with @holisticode after asking you guys from core :+1: